libstuff/STCPNode.cpp

```diff
@@ -165,7 +164,7 @@ void STCPNode::postSelect(fd_map& fdm, uint64_t& nextActivity) {
                             // We set a lower bound on this at 1, because even though it should be pretty impossible
                             // for this to be 0 (it's in us), we rely on it being non-zero in order to connect to
                             // peers.
-                            peer->latency = std::max(STimeNow() - message.calc64("Timestamp"), UINT64_C(0x1));
+                            peer->latency = max(STimeNow() - message.calc64("Timestamp"), UINT64_C(0x1));
```

Comment:
Oh, and I just noticed -- can't you just do something simpler like `(uint64_t)1` rather than that weird macro and a hex value?